lib/rucio/rse/protocols/webdav.py

```diff
@@ -516,6 +516,7 @@ def stat(self, path):
             if file.size is None:
                 continue
 
+            dict_['adler32'] = file.href
```

Comment:
href doesn't seem to be the adler32 checksum though? That seems to "conflict" with how href is used above?